taxonomy.simple.yaml

```diff
@@ -30,30 +30,31 @@
       - Emergency Food
       - Temporary Shelter:
         - Weather Relief
-      - Help Find Missing Persons
+        - Help Find Missing Persons
```

Comment:
This seems too indented..

Reply:
Oh yes, my mistake. Fixing.

Reply:
I committed another change to this branch so should be fixed now